pkg/controller/gkenetworkparamset/gkenetworkparamset_metrics.go

```diff
@@ -20,13 +20,14 @@ var (
 		},
 		[]string{"networkName", "status", "type"},
 	)
-	fetchSubnetErrs = metrics.NewCounter(
```

Comment:
Removing fetch subnet errors because its already exported in from the GCE client in this repository. This metric is completely redundant.